src/sass/modules/_m-schemaform.scss

```diff
@@ -284,7 +284,7 @@ label {
   }
   > .schemaform-file-item-edit {
     flex-direction: column;
-    align-items: left;
```

Comment:
I don't think it would help in this case since I just messed up a property value, but that seems like a really useful tool.